frontend/js/components/organisms/TextField.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class TextField extends Component {
+    render() {
+        return (
+            <div className="text-field">
+                {
+                    this.props.label &&
+                    <label htmlFor={this.props.name}>
+                        {this.props.label}
+                    </label>
+                }
+                <div className={`text-field-input ${this.props.button && 'has-button'}`}>
```

Comment:
This is checking to see if `this.props.button` is truthy, and if so it returns the string `has-button`.